modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -70,13 +71,13 @@ public String getHelp() {
 
     @Override
     public boolean isDataTypeFile(final String dataType) {
-        return "Buffer".equals(dataType);
+        return dataType != null && (dataType.equals("Buffer") || dataType.equals("FormDataFile"));
     }
 
     @Override
     public String getTypeDeclaration(Schema p) {
         if (ModelUtils.isFileSchema(p)) {
-            return "Buffer";
+            return "Buffer|FormDataFile";
```

Comment:
I think this also affects the api *response* type, so it would be a breaking change, right?